### ISSUE TITLE
Finish up fos 2.x pull-request

### ISSUE THREAD

**Issue by pulse00** [pulse00/SonataUserBundle#1]
@DavidSoussan I've added you to this repo, so you should be able to commit. The branch is `fos-2.x`

**Comment by pulse00** [comment]
Here's what i had in mind to do until opening a PR:
- [ ] Update documentation regarding fos 1/2.x usage
- [ ] Fix tests

**Comment by pulse00** [comment]
@VolodymyrChornohal you should have commit access too now. Thanks for helping!

**Comment by nmathon** [comment]
Is it working with Group management for you ?
I have this error : 

```
Attempted to load class "GroupManager" from namespace "FOS\UserBundle\Entity".
Did you forget a "use" statement for another namespace?
```

And effectively, there is still some old namespaces in GroupManager.

Nicolas

**Comment by nmathon** [comment]
@pulse00 I corrected things and now it works for me. Can you give me commit access or do you prefer I create a fork + PR ?

**Comment by pulse00** [comment]
@nmathon sorry i was out of town for a couple of days. i've added you to the collaborators.

**Comment by EnoahNetzach** [comment]
@nmathon I don't know what you did for the GroupManager problem, but I found out it's only a little namespace problem in _sonata-project/user-bundle/Entity/GroupManager.php_ line 14; `FOS\UserBundle\Doctrine\GroupManager as BaseGroupManager:` is the correct one.

Also is it me, or the v1 controllers have to be commented (or deleted), because I get

```
Runtime Notice: Declaration of Sonata\UserBundle\Controller\SecurityFOSUser1Controller::loginAction() should be compatible with FOS\UserBundle\Controller\SecurityController::loginAction(Symfony\Component\HttpFoundation\Request $request) in [...]/sonata-project/user-bundle/Controller/SecurityFOSUser1Controller.php line 40
```

**Comment by menzow** [comment]
Any updates of this being merged into SonataUserBundle master?

**Comment by pulse00** [comment]
@menzow yeah, sorry i didn't mention it, i've opened a PR here https://github.com/sonata-project/SonataUserBundle/pull/555

**Comment by menzow** [comment]
Thanks, I'll follow that :)
